README.md

```diff
@@ -231,18 +231,28 @@ Install using [`lazy.nvim`](https://github.com/folke/lazy.nvim):
 
 ```lua
 -- nvim v0.8.0
-require("lazy").setup({
-    {
-        "kdheepak/lazygit.nvim",
-        dependencies =  {
-            "nvim-telescope/telescope.nvim",
-            "nvim-lua/plenary.nvim"
-        },
-        config = function()
-            require("telescope").load_extension("lazygit")
-        end,
+ {
+    "kdheepak/lazygit.nvim",
+    cmd = {
+      "LazyGit",
+      "LazyGitConfig",
+      "LazyGitCurrentFile",
+      "LazyGitFilter",
+      "LazyGitFilterCurrentFile",
     },
-})
+    config = function()
+      require("lazy").setup({})
+    end,
```

Comment:
I'm pretty sure this is not required, right? This config is if `lazygit` needs a setup function which it does not at the moment.

Reply:
Yes both 
```lua  
{
    "kdheepak/lazygit.nvim",
    cmd = {
      "LazyGit",
      "LazyGitConfig",
      "LazyGitCurrentFile",
      "LazyGitFilter",
      "LazyGitFilterCurrentFile",
    },
    -- optional for floating window border decoration
    dependencies = {
      "nvim-lua/plenary.nvim",
    },
    -- setting the keybinding for LazyGit with 'keys' is recommended in
    -- order to load the plugin when the command is run for the first time
    keys = {
      { "<leader>gg", "<cmd>LazyGit<cr>", desc = "LazyGit" }
    }
  },
```
and 

```lua
    "kdheepak/lazygit.nvim",
    lazy = false,
    cmd = {
      "LazyGit",
      "LazyGitConfig",
      "LazyGitCurrentFile",
      "LazyGitFilter",
      "LazyGitFilterCurrentFile",
    },
    -- optional for floating window border decoration
    dependencies = {
      "nvim-lua/plenary.nvim",
    },
```
works. I didn't know that lazygit.nvim didn't have a setup function.